docker/falco/versions.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Google LLC
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+
+cloudbuild:
+  enable_parallel: false
+versions:
+- dir: 0/debian11/0.31
+  from: marketplace.gcr.io/google/debian11
+  packages:
+    falco:
+      version: 0.31.1
+  repo: falco
+  tags:
+  - 0.31.1-debian11
+  - 0.31-debian11
+  - 0-debian11
+  - 0.31.1
+  - '0.31'
+  - '3'
+  - latest
+
+- dir: exporter
+  excludeTests:
+  - tests/functional_tests/functional_test.yaml
+  from: marketplace.gcr.io/google/debian11
+  packages:
+    falco_exporter:
+      sha256: cda8d6c60c240c73c28ed0d4e81cb7b52808dcf0358aec41fddc93a30d1e68b7
+      version: 0.7.0
+  repo: falco
```

Comment:
Let's create another container repo for exporter. Check postgresql-exporter case: https://github.com/GoogleCloudPlatform/postgresql-docker/blob/master/versions.yaml#L149

It will be make it easier for automate the upgrades.